data/Makefile

```diff
@@ -641,6 +641,15 @@ minsize=2000
 %.histo: %.hist
 	sed -n 's/^f//p' $< | tr '\t' ' ' >$@
 
+# Count k-mers using ntCard.
+%_k$(k).histogram.txt: %.fq.gz
+	$(time) ntcard -t $(t) -k $(k) -o $@ $<
```

Comment:
This is picky, but to keep it consistent -- we generally use `$k` for single-letter variables, and only the parentheses with multi-letter variables (ex. `$(time)`)